### ISSUE TITLE
Add a bot for closing duplicate PRs and eventually add some automation

### ISSUE THREAD

**Issue by goanpeca** [jupyterlab/jupyterlab#8658]
From the discussion over at https://github.com/jupyterlab/jupyterlab/issues/8117#issuecomment-654890148, Github actions currently do not support closing Duplicate PRs on forks. They can be closes on branches in the same repo with different actions available on the market place, but for forks, it poses a problem as every commit a user makes will trigger the build chain, which may end up draining the available github API builder rates.

Following on the work with conda forge, I suggest creating a similar Heroku app to close duplicate issues on forks, so that no stale builds remain to consume. I already have something that does this and created this issue to track the work and see if there are any other possibilities for automating other processes.

Cheers!

**Comment by blink1073** [comment]
Thanks!  Do you have a link to the conda forge work?

**Comment by goanpeca** [comment]
https://github.com/conda-forge/conda-forge-webservices

**Comment by blink1073** [comment]
Nice, thank you

**Comment by blink1073** [comment]
(oh, that repo is private btw, I happen to still have access)

**Comment by goanpeca** [comment]
(I know ;-)., but you get the idea) I could set this up pretty quickly. Just need a repo for it to connect to Heroku and set the token for a bot account. Is there any bot account on Jlab?

**Comment by blink1073** [comment]
No, we have not ever done one, and I have never used Heroku ;-)

**Comment by goanpeca** [comment]
>No, we have not ever done on

Maybe it is a good time :-) 

>and I have never used Heroku ;-)

I can set it up and ping you, it is pretty straight forward. Since it all is handled with a docker image the only thing that needs to be setup in heroku is the env variables with the tokens.
